emma-language/src/test/scala/org/emmalanguage/compiler/BaseCompilerSpec.scala

```diff
@@ -50,20 +50,25 @@ trait BaseCompilerSpec extends FreeSpec with Matchers with PropertyChecks with T
   }
 
   lazy val idPipeline: u.Expr[Any] => u.Tree = {
-      (_: u.Expr[Any]).tree
-    } andThen {
-      compiler.identity(typeCheck = true)
-    } andThen {
-      checkCompile
-    }
+    (_: u.Expr[Any]).tree
+  } andThen {
+    compiler.identity(typeCheck = true)
+  } andThen {
+    checkCompile
+  }
```

Comment:
I suggest moving `_.tree` to the end to avoid specifying the parameter type again:
```scala
lazy val idPipeline: u.Expr[Any] => u.Tree = {
  compiler.identity(typeCheck = true)
} andThen {
  checkCompile
} compose {
  _.tree
}
```